src/Datadog.Trace.ClrProfiler.Native/Resource.rc

```diff
@@ -70,7 +70,7 @@ BEGIN
             VALUE "FileDescription", "Datadog CLR Profiler"
             VALUE "FileVersion", "1.18.2.0"
             VALUE "InternalName", "Datadog.Trace.ClrProfiler.Native.DLL"
-            VALUE "LegalCopyright", "Copyright (C) 2017-2019"
+            VALUE "LegalCopyright", "Copyright (C) Datadog, Inc. 2017"
```

Comment:
I don't think the copyright mention makes sense since the code is published under APACHE license. But assuming it does, we should use the year range: https://www.copyrightlaws.com/copyright-symbol-notice-year/

> New versions or editions of works should contain the publication date of the new version or edition. For constantly evolving websites and blogs that contain works published over several years, the notice may include a range of years (e.g., 2011–2020), starting from the date of the oldest published elements and ending with the date of the newest published elements.

Reply:
One of the main conditions of Apache License 2.0 require preservation of copyright ([source](https://choosealicense.com/licenses/apache-2.0/)). The license text even includes a template for the copyright notice in its appendix:
```
APPENDIX: How to apply the Apache License to your work.

      To apply the Apache License to your work, attach the following
      boilerplate notice...

   Copyright [yyyy] [name of copyright owner]
```

Regarding the year range, the text you quoted says "the notice **may** include a range of years", so it doesn't sound required. Our Open Source team recommends using `2017` or `2017-Present` so we don't need to keep updating it.